docs/styling.md

```diff
@@ -3,15 +3,16 @@ category: Documentation
 categoryindex: 1
 index: 7
 ---
-# Customization and Styling 
+
```

Comment:
Please revert all changes to this file that are not new content. If you want to clean up something you can do this in another PR. The goal of this one is to add a custom menu structure, any other change is just confusing noise. The git history will be confusing because of this.
You always want to make a PR as small as possible. Provide minimum changes to solve the problem and don't touch anything else.

Reply:
Sorry Nojaf, this one got a little out of hand. I don't know why my IDE made all of those changes.